massgov/mayflower
# DP-16029 See all link style

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Change the property value to set style for see all link in service page to match Mass.gov production by applying the correct set of class to the element.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-16029)


## Steps to Test


1. Go to http://localhost:3000/patterns/05-pages-service/05-pages-service.html.

2. Find the see all links in the **What would you like to do?** and the **What you need to know** sections match the style in the [production one](https://www.mass.gov/qag-service4).

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
